packages/trip-form/src/settings-selector-panel.story.js

```diff
@@ -29,3 +30,12 @@ export const settingsSelectorPanel = () => (
 );
 export const settingsSelectorPanelStyled = () =>
   trimet(settingsSelectorPanel());
+
+export const settingsSelectorPanelUndefinedParams = () => (
+  <SettingsSelectorPanel
+    queryParams={queryParams}
+    supportedModes={commonModesEmpty}
+    supportedCompanies={undefined}
```

Comment:
Not sure it's necessary to set supportedCompanies to `undefined`, but I don't have a strong feeling on this.